src/v2/Apps/Partner/Components/PartnerArtists/partnerArtistsUtils.ts

```diff
@@ -0,0 +1,95 @@
+import { PartnerArtistList_artists } from "v2/__generated__/PartnerArtistList_artists.graphql"
+
+export interface ArtistListColumnSizes {
+  representedArtist: number
+  worksAvailableBy: number
+}
+
+export function getColumnsCountForEachArtistType(
+  representedArtistCount: number,
+  worksAvailableByCount: number,
+  columnCount: number
+): ArtistListColumnSizes {
+  if (representedArtistCount === 0 && worksAvailableByCount === 0) {
+    return {
+      representedArtist: 0,
+      worksAvailableBy: 0,
+    }
+  }
+
+  const autistsInColumn = Math.ceil(
```

Comment:
Shoot missed this one too-- should be `artistsInColumn`

Reply:
Fixed